frontend/src/scenes/notebooks/Notebook/notebookLogic.ts

```diff
@@ -48,8 +48,9 @@ export const notebookLogic = kea<notebookLogicType>([
     path((key) => ['scenes', 'notebooks', 'Notebook', 'notebookLogic', key]),
     key(({ shortId }) => shortId),
     connect({
-        values: [notebooksModel, ['scratchpadNotebook', 'notebookTemplates']],
-        actions: [notebooksModel, ['receiveNotebookUpdate']],
+        // TODO: Why does uncommenting this break things
```

Comment:
For me uncommenting these lines causes kea to think the "notebooksModel" is undefined...

Reply:
connect needs to take a function not an object when this happens 🙉